### ISSUE TITLE
Assertion '!modifiers.empty()' when running gamescope in embedded mode

### ISSUE THREAD

**Issue by Erfboom** [ValveSoftware/gamescope#1215]
Using the following:

 - Intel UHD630
 - 6.8.1-1-default linux kernel
 - Opensuse Tumbleweed
 - `gamescope -W 2560 -H 1440 -r 120 -- moonlight` in tty2 (embedded)
 
 I receive `Assertion '!modifiers.empty()'` after a core dump. The program can be anything, not just moonlight.
 
 When I run gamescope in nested mode, I can run things, but for moonlight specifically, once I activate a stream, I'll see a black frame in between every couple of good frames. If I use `--expose-wayland`, this will be reduced, but I'll trade it for choppiness in the stream on the native moonlight-qt, or smoothness with the AppImage variant but still the black frames.
 
 If I run moonlight from terminal, no gamescope, the stream is as intended, smooth, no black frames.

**Comment by rien** [comment]
I'm encountering a segfault as well on an Intel UHD620 - @Erfboom did you manage to work around this issue?

**Comment by Erfboom** [comment]
No, but I believe it's being worked on.

**Comment by jpgrego** [comment]
Same problem here, also with a HD620. Any news regarding this?

**Comment by repomansez** [comment]
Happening to me too. UHD Graphics 620 on a ThinkPad T480

**Comment by 2goodAP** [comment]
Yes, I too can reproduce this on NixOS with Intel GPU. I am using KDE Plasma 6.1.4. Also, the assertion failure only happens in nested mode. Everything works just fine for me under a separate `tty`.

I did some digging around and found this to be an issue for 3.15.x versions only. Nested mode works just fine for me up to commit 94271f3.

The 3.15.0 commit 28a4744, seems to have made some changes to DRM formats for _composited outputs_. That seems to be a possible cause for the assertion failure in nested mode only.

**Comment by asukaminato0721** [comment]
it also failed on

GPU: Intel Iris Xe Graphics @ 1.30 GHz [Integrated]

**Comment by repomansez** [comment]
Upgrading to 3.15.9 completely fixed the issue for me. @jpgrego @2goodAP @asukaminato0721 @Erfboom  can you guys try to update and see if it works too? Gentoo and NixOS specifically are shipping an out of date version of Gamescope.

**Comment by 2goodAP** [comment]
Yeah 3.15.9 seems to have fixed it for me too. Thanks for the heads up @repomansez.

**Comment by asukaminato0721** [comment]
arch updated to

```
[gamescope] [Info]  console: gamescope version 3.15.11 (gcc 14.2.1)
```

and it fixed for me too. Thanks for the heads up @repomansez.
